datafusion/core/tests/parquet/row_group_pruning.rs

```diff
@@ -578,6 +578,27 @@ async fn prune_decimal_in_list() {
         6,
     )
     .await;
+    // test data -> r1: {1,2,3,4,5}, r2: {1,2,3,4,6}, r3: {1,2,3,4,6}
+    test_row_group_prune(
+        Scenario::DecimalBloomFilter,
+        "SELECT * FROM t where decimal_col in (5)",
+        Some(0),
+        Some(0),
+        Some(2),
```

Comment:
`Some(2),` is bloom filter pruned, will make more clearly in https://github.com/apache/arrow-datafusion/issues/8922

Reply:
Related PR https://github.com/apache/arrow-datafusion/pull/8941